docs(hcp): add self-service user account deletion docs and changelog (#2865)
hashicorp/web-unified-docs: content/hcp-docs/content/docs/changelog.mdx

# Changelog

### 2026-07-20

**HCP user account deletion:** Users that log in through GitHub or with their email and password can now delete their own HCP accounts directly from the portal without filing a support ticket. Refer to [Delete your account](/hcp/docs/hcp/create-account#delete-your-account) for instructions and other details. 

For more information, refer to [Delete your account](/hcp/docs/hcp/create-account#delete-your-account).

> 

Reviewer: ```suggestion
```
Repeat of the previous line

> **HCP user account deletion:** GitHub and email/password users can now delete their own HCP accounts directly from the portal, without filing a support ticket. The self-service flow includes an automated pre-flight check that blocks deletion if you are the owner of an HCP organization. If blocked, you can resolve the dependency by transferring ownership to another user or deleting the organization, then retry deletion. SSO and SCIM-provisioned users are not eligible for self-service deletion and must contact their organization administrator to be deprovisioned through their identity provider.

Reviewer 2: ```suggestion
**HCP user account deletion:** GitHub and email/password users can now delete their own HCP accounts directly from the portal, without filing a support ticket. The self-service flow includes an automated pre-flight check that blocks deletion if you are the owner of an HCP organization. If you are blocked, you can resolve the dependency by transferring ownership to another user or deleting the organization, then retrying deletion. SSO and SCIM-provisioned users are not eligible for self-service deletion and must contact their organization administrator to be deprovisioned through their identity provider.
```

> **HCP user account deletion:** GitHub and email/password users can now delete their own HCP accounts directly from the portal, without filing a support ticket. The self-service flow includes an automated pre-flight check that blocks deletion if you are the owner of an HCP organization. If you are blocked, you can resolve the dependency by transferring ownership to another user or deleting the organization, then retrying deletion. SSO and SCIM-provisioned users are not eligible for self-service deletion and must contact their organization administrator to be deprovisioned through their identity provider.

Reviewer: ```suggestion
**HCP user account deletion:** Users that log in through GitHub or with their email and password can now delete their own HCP accounts directly from the portal without filing a support ticket. Refer to [Delete your account](/hcp/docs/hcp/create-account#delete-your-account) for instructions and other details. 
```
Too much detail for the changelog. FWIW, the entry for SCIM support is also TMI.

Author: Ill review the SCIM one too.
